packages/core/contracts/financial-templates/common/financial-product-libraries/StructureNoteFinancialProductLibrary.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "../../../common/implementation/Testable.sol";
+
+import "./FinancialProductLibrary.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+
+interface ExpiringMultiParty {
```

Comment:
Should this contract be in a test folder?

Reply:
I don't think so. It's not a test. It's an actual Financial product library that implements a desired kind of product. This is why I put it in this new directory `financial-product-libraries`. I invasion we'll have a collection of different libraries in this directory to do a bunch of custom overrides, this is just the first for this.